File: app/src/main/java/com/reddittask/android/util/DateUtil.java
package com.reddittask.android.util;

import android.text.format.DateUtils;

import java.text.Format;
import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.util.Calendar;
import java.util.Date;
import java.util.Locale;
import java.util.TimeZone;

/**
 * Created by marinaracu on 30.11.2017.
 */

public class DateUtil {

    public static String convertTime(int time) {
        long timestamp = time;
        Calendar cal = Calendar.getInstance();
        TimeZone tz = cal.getTimeZone();
        CharSequence relTime = DateUtils.getRelativeTimeSpanString(
                timestamp * 1000,
                System.currentTimeMillis(),
                DateUtils.MINUTE_IN_MILLIS);

        return relTime.toString();
    }
}

File: app/src/main/java/com/keddits/util/DateUtil.kt
package com.keddits.util

import android.text.format.DateUtils
import java.util.*

/**
 * Created by marinaracu on 04.12.2017.
 */
class DateUtil {
    companion object {
        fun convertTime(time: Int): String {
            val timeStamp: Long = time.toLong()
            val cal: Calendar = Calendar.getInstance()
            val timeZone: TimeZone = cal.timeZone
            val realTime: CharSequence = DateUtils.getRelativeTimeSpanString(
                    timeStamp * 1000,
                    System.currentTimeMillis(),
                    DateUtils.MINUTE_IN_MILLIS)
            return realTime.toString()
        }
    }
}